datasets/uy/pep/ur_pep.yml

```diff
@@ -0,0 +1,14 @@
+name: uy_pep
+title: "Politically Exposed Persons (PEP)"
+url: https://www.gub.uy/
+description: placeholder
+entry_point: crawler.py
+prefix: ur-pep
```

Comment:
let's name this file uy_pep and make the prefix uy-pep so that it all agrees on the country prefix